lib/src/http_client_response.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+part of '../src/http_client_request.dart';
+
+abstract class HttpClientResponse extends Stream<List<int>> {
+  HttpClientResponse();
+
+  factory HttpClientResponse._(Stream<List<int>> cbhStream) {
+    return _HttpClientResponse(cbhStream);
+  }
+}
+
+class _HttpClientResponse extends HttpClientResponse {
```

Comment:
You should consider adding dart docs to internal classes as well, this will be helpful in the future. There have been countless occasions when I didn't remember the purpose of a class in ffigen but even a single-line description made it easier.

It is okay to leave the dart doc if it's very trivial. So this is just a suggestion you can use accordingly.

Reply:
Added :)